src/main/msp/msp_serial.c

```diff
@@ -37,8 +37,12 @@
 #include "msp/msp.h"
 #include "msp/msp_serial.h"
 
-static mspPort_t mspPorts[MAX_MSP_PORT_COUNT];
+#include "msp/msp_protocol.h"
 
+#include "config/parameter_group.h"
+#include "config/parameter_group_ids.h"
+static mspPort_t mspPorts[MAX_MSP_PORT_COUNT];
```

Comment:
Might be a good idea to add a line return between the `#include` block and the static object here?